mqtt/mqtt-broker/src/connection.rs

```diff
@@ -264,17 +266,20 @@ where
                     Packet::PubAck(puback) => ClientEvent::PubAck(puback),
                     Packet::PubComp(pubcomp) => ClientEvent::PubComp(pubcomp),
                     Packet::Publish(publish) => {
+                        #[cfg(feature = "edgehub")]
                         let publish = translate_incoming_publish(&client_id, publish);
                         ClientEvent::PublishFrom(publish)
                     }
                     Packet::PubRec(pubrec) => ClientEvent::PubRec(pubrec),
                     Packet::PubRel(pubrel) => ClientEvent::PubRel(pubrel),
                     Packet::Subscribe(subscribe) => {
+                        #[cfg(feature = "edgehub")]
                         let subscribe = translate_incoming_subscribe(&client_id, subscribe);
                         ClientEvent::Subscribe(subscribe)
                     }
                     Packet::SubAck(suback) => ClientEvent::SubAck(suback),
                     Packet::Unsubscribe(unsubscribe) => {
+                        #[cfg(feature = "edgehub")]
```

Comment:
q: what is the scope of `cfg`, just a following expression? It's hard to find the answer online :(

Reply:
Correct. To wrap more than one expression one need to wrap it with `{...}`